torch/_inductor/fx_passes/post_grad.py

```diff
@@ -898,9 +898,14 @@ def cat_noop(inputs, dim=0):
     return len(inputs) == 1
 
 
-@register_noop_decomp(aten.view)
-def view_noop(arg, size):
-    return arg.shape == size
+@register_noop_decomp(aten.view.default)
+def view_default_noop(arg, size):
+    return statically_known_true(sym_eq(arg.shape, tuple(size)))
```

Comment:
arg.shape is a tuple but not a list. [`sym_eq`](https://github.com/pytorch/pytorch/pull/113159#discussion_r1385323774) returns False if one input is tuple and another input is list.

Reply:
Did this used to specialize on size?